source/manual/covid-19-services.html.md

```diff
@@ -0,0 +1,330 @@
+---
+owner_slack: "#govuk-corona-forms-tech"
+title: COVID-19 Services
+section: Services
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-03-25
+review_in: 1 month
+---
+
+We recently launched 2 new services:
+
+- Offer coronavirus support from your business
+- Get coronavirus support as an extremely vulnerable person
+
+There is also an interactive voice response (IVR) automated phone
```

Comment:
It appears TechOps maintains this, ongoing chat in slack about if they have any docs we can link to.